app/test_main.py

```diff
@@ -1,3 +1,30 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    def test_should_be_equal_zero(self):
+        # if age less than 15
+        assert get_human_age(0, 0) == [0, 0]
+        assert get_human_age(3, 2) == [0, 0]
+        assert get_human_age(12, 4) == [0, 0]
+        assert get_human_age(14, 14) == [0, 0]
+
+    def test_should_be_equal_one(self):
+        # if animal age less than summ first year and second year but more than first year
+        assert get_human_age(17, 17) == [1, 1]
+        assert get_human_age(18, 2) == [1, 0]
+        assert get_human_age(19, 23) == [1, 1]
+        assert get_human_age(2, 23) == [0, 1]
```

Comment:
These test cases could be combined using `@pytest.mark.parametrize` to reduce repetition and improve clarity.